pkg/backends/calico/routes.go

```diff
@@ -348,6 +348,12 @@ func contains(items []string, target string) bool {
 func (rg *routeGenerator) advertiseThisService(svc *v1.Service, ep *v1.Endpoints) bool {
 	logc := log.WithField("svc", fmt.Sprintf("%s/%s", svc.Namespace, svc.Name))
 
+	// Don't advertise routes if this node is explicitly excluded from load balancers.
+	if rg.client.ExcludeServiceAdvertisement() {
+		logc.Debugf("Skipping service because node is explicitly excluded from load balancers %s", svc.Spec.Type)
```

Comment:
Probably don't need the service type in this log message as it is.

Actually maybe the `svc.Spec.Type` could be embedded in the `logc` Entry and the other log statements could be simpler.